torch/testing/_internal/opinfo/definitions/fft.py

```diff
@@ -823,10 +823,18 @@ def mt(shape, **kwargs):
         "_refs.fft.fftshift",
         op_db=op_db,
         torch_opinfo_name="fft.fftshift",
+        skips=(
+            # TODO Move fftshift to decomps
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref'),
+        ),
     ),
     PythonRefInfo(
         "_refs.fft.ifftshift",
         op_db=op_db,
         torch_opinfo_name="fft.ifftshift",
+        skips=(
+            # TODO Move ifftshift to decomps
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref'),
+        ),
```

Comment:
@peterbell10 I had to add these. Do you want me to append a PR moving these ops to decomps and removing these xfails, or do you prefer to leave them like this?

Reply:
Seems fine to keep it as a skip imo